nvtabular/ops/categorify.py

```diff
@@ -649,22 +680,22 @@ def _write_uniques(dfs, base_path, col_group, on_host, concat_groups, name_sep):
         new_cols = {}
         nulls_missing = False
         for col in col_group:
-            if not df[col]._column.has_nulls:
+            if not _series_has_nulls(df[col]):
                 nulls_missing = True
                 new_cols[col] = _concat(
-                    [cudf.Series([None], dtype=df[col].dtype), df[col]],
+                    [df._constructor_sliced([None], dtype=df[col].dtype), df[col]],
```

Comment:
I had no idea about this 'constructor_sliced' property - neat!

Reply:
Right?  I had two pandas devs confirm that this is the best way to do this when I was working on dask shuffle :)